sys-libs/readline/readline-8.2.013.recipe

```diff
@@ -10,20 +10,33 @@ The history facilites are also placed into a separate library, the History \
 library, as part of the build process. The History library may be used without \
 Readline in applications which desire its capabilities."
 HOMEPAGE="http://tiswww.case.edu/php/chet/readline/rltop.html"
-COPYRIGHT="1989-2019 Free Software Foundation, Inc."
+COPYRIGHT="1989-2024 Free Software Foundation, Inc."
 LICENSE="GNU GPL v3"
 REVISION="1"
 SOURCE_URI="ftp://ftp.gnu.org/gnu/readline/readline-8.2.tar.gz"
 SOURCE_DIR="readline-8.2"
-for i in 001; do
+for i in 013; do
```

Comment:
This requires a sequence, like in the deleted recipes:

```suggestion
for i in {001..013}; do
```

Your code would only use patch number 013, not the other ones before.